WORKSPACE

```diff
@@ -41,8 +41,14 @@ load("//library/maven:rules.bzl", "maven")
 # Load //builder/kotlin
 load("//builder/kotlin:deps.bzl", kotlin_deps = "deps")
 kotlin_deps()
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
-kotlin_repositories()
+load("@io_bazel_rules_kotlin//kotlin:repositories.bzl", "kotlin_repositories", "kotlinc_version")
+kotlin_repositories(
+    compiler_release = kotlinc_version(
+        release = "1.7.20",
+        sha256 = "5e3c8d0f965410ff12e90d6f8dc5df2fc09fd595a684d514616851ce7e94ae7d"
+    )
+)
+load("@io_bazel_rules_kotlin//kotlin:core.bzl", "kt_register_toolchains")
```

Comment:
Since we forked, `rules_kotlin` has adopted a more modular approach for functions rather than having everything be imported from (or aliased through) `kotlin:kotlin.bzl`.